doc/source/citing.rst

```diff
@@ -0,0 +1,25 @@
+Citing SNEWPY
+=============
+
+If you use SNEWPY for your own research, please cite the following two papers:
+
+* \A. L. Baxter `et al.` (SNEWS Collaboration):
+  “SNEWPY: A Data Pipeline from Supernova Simulations to Neutrino Signals”.
+  Journal of Open Source Software 6 (2021) 67, 3772.
+  [`DOI:10.21105/joss.03772 <https://dx.doi.org/10.21105/joss.03772>`_]
+
+* \A. L. Baxter `et al.` (SNEWS Collaboration):
+  “SNEWPY: A Data Pipeline from Supernova Simulations to Neutrino Signals”.
+  Astrophysical Journal (2022), in print.
+  [`arXiv:2109.08188 <https://arxiv.org/abs/2109.08188>`_, `DOI:10.3847/1538-4357/ac350f <https://dx.doi.org/10.3847/1538-4357/ac350f>`_]
```

Comment:
This DOI doesn't resolve (yet) - is this a future DOI reference?

Reply:
Yep, that will be the DOI of the ApJ paper once it’s published—see https://github.com/openjournals/joss-reviews/issues/3772#issuecomment-956288092 (and the JOSS paper itself). Since we’ve returned the proof to them last week, it shouldn’t take much longer until the paper gets published.